src/tests/page_objects/planner/planner.page.ts

```diff
@@ -3,6 +3,7 @@ import { v4 as uuid } from 'uuid';
 import { AppPage } from '../app.page';
 import * as planner from './../../ui/planner';
 import * as support from './../../support';
+import { Key } from 'selenium-webdriver';
```

Comment:
We should use http://www.protractortest.org/#/api?view=webdriver.WebElement.prototype.sendKeys instead of `key` from selenium-webdriver. We don't import selenium-webdriver's API, we use protractor's API that internally uses selenium APIs.